src/nvim/memory.h

```diff
@@ -2,6 +2,104 @@
 #define NVIM_MEMORY_H
 
 #include <stddef.h>  // for size_t
+#include <string.h>
+#include <strings.h>
```

Comment:
Right, it's for `strcasecmp` only. I could wrap it with `HAVE_STRINGS_H`, but I chose `HAVE_STRCASECMP` instead since it seems more topical.